src/node/rpc/test/frontend_test.cpp

```diff
@@ -486,7 +486,7 @@ void add_callers_pbft_store()
   CHECK(g.finalize() == kv::CommitSuccess::OK);
 }
 
-TEST_CASE("process_pbft")
+TEST_CASE("process_bft")
 {
   add_callers_pbft_store();
   TestUserFrontend frontend(*pbft_network.tables);
```

Comment:
```suggestion
  add_callers_bft_store();
  TestUserFrontend frontend(*bft_network.tables);
```

NB: These would need renaming throughout the file, but I think they should be.